sota/routers/audient_router.py

```diff
@@ -7,7 +10,47 @@
     tags=["audient"]
 )
 
+class RequestAudientData(BaseModel):
+    country_code: str
+    sport_id: List[int]
+    gender: str
+    age: int
+
+    @validator("gender")
+    def validate_gender(cls, value):
+        allowed_values = {"M", "F", "N"}
+        if value not in allowed_values:
+            raise ValueError("Invalid value for gender. It must be 'M', 'F', or 'N'.")
+        return value
+
+class RequestListOfAudientData(BaseModel):
+    audience: List[RequestAudientData]
+
+def convert_data(data):
+    return{
+        "country_code": data["country_code"],
+        "sport_id": data["sport_id"],
+        "gender": data["gender"],
+        "age": data["age"]
+    }
 
 @router.get("/")
 def get_audient():
-    return {"test": "all good"}
+    audient_all = [convert_data(audient) for audient in audient_collection.find()]
+    return audient_all
+
+@router.post("/update_audient_info")
+async def update_audient_info(data: RequestListOfAudientData):
+    audient_id = 0
```

Comment:
Why do you declare `audient_id` here as 0?

Reply:
Because if there no data before we need to assign _id to data before insert so I use audient_id = 0 if there no data before

Reply:
Can we use `None` instead?

Reply:
No because NoneType can't + with int

Reply:
I use audient_id in loop to assign _id to each data

Reply:
Resolving this conversation since I forgot to update the API draft, 😜